tests/sentry/issues/test_escalating.py

```diff
@@ -17,15 +17,15 @@
 from sentry.models import Group
 from sentry.models.group import GroupStatus
 from sentry.models.groupinbox import GroupInbox
-from sentry.testutils import SnubaTestCase, TestCase
+from sentry.testutils import TestCase
 from sentry.types.group import GroupSubStatus
 from sentry.utils.cache import cache
 from sentry.utils.snuba import to_start_of_hour
 
 TIME_YESTERDAY = (datetime.now() - timedelta(hours=24)).replace(hour=6)
 
 
-class BaseGroupCounts(SnubaTestCase):  # type: ignore[misc]
+class BaseGroupCounts(TestCase):  # type: ignore[misc]
```

Comment:
You need `SnubaTestCase` here as well if you're using snuba in these tests. Otherwise we won't reset snuba afterwards and the tests will be unstable

Reply:
That's good to know.

I will follow up on this.

Reply:
#48330 fixes the problem with `SnubaTestCase` not executing.

#48332 is the follow up to your comment.